drv/sidecar-front-io/src/phy_smi.rs

```diff
@@ -45,15 +45,12 @@ impl PhySmi {
     }
 
     #[inline]
-    pub fn phy_power_enabled(&self) -> Result<bool, FpgaError> {
+    pub fn power_enabled(&self) -> Result<bool, FpgaError> {
```

Comment:
Why rename these?  I'd be worried about potential confusion with front IO HSC control; the old name is nicely explicit that this is PHY power control.

Reply:
I'm inclined to agree with Matt here, but presumably this was done since the functions are within the `phy_smi` module so adding `phy` into all the names is a bit redundant. I tend to have more verbose tendencies with naming though.

Reply:
Yeah I am all about localized naming. But if both of you like having the phy portion in there I'll put it back.